test/features/deploy_kogito_runtime.feature

```diff
@@ -351,18 +351,18 @@ Feature: Deploy Kogito Runtime
 
     Then Service "process-mongodb" contains 1 instances of process with name "dealreviews" within 2 minutes
 
-    @springboot
+  @springboot
```

Comment:
The previous annotation alignment was correct one, can you please return it back?